mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -595,7 +595,9 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
                   b.getNamedAttr("parameters",
                                  b.getArrayAttr({
                                      b.getI32IntegerAttr(leftPadH),
+                                     b.getI32IntegerAttr(rightPadH),
```

Comment:
How about backward weight and backward data?

Reply:
this logic is the same with original code,
bwd and wrw do transform on this attr